src/messaging.js

```diff
@@ -209,6 +209,10 @@ var async = require('async'),
 			};
 
 			plugins.fireHook('filter:messaging.parse', messageData, function(err, messageData) {
+				if (err) {
```

Comment:
How do you know that the error always indicates that a file is missing? Also if an error exists I would assume that `messageData` is `null` or `undefined`, so accessing the `parsedMessage` property would fail.